docs/coordinates/representations.rst

```diff
@@ -314,6 +314,138 @@ provides the ``base``::
   <SphericalRepresentation (lon, lat, distance) in (rad, rad, kpc)
       ( 0.0048481,  1.04719246,  1.00000294)>
 
+.. Note:: At present, the differential classes are generally meant to work with
+   first derivatives, but they do not check the units of the inputs to enforce
+   this. Passing in 2nd derivatives, e.g., acceleration values (with
+   acceleration units), will succeed, but any transformations that occur through
+   re-representation of the differential will not necessarily be correct.
```

Comment:
Is this true? I think they will be correct (at the representation level at least). They certainly are correct for offsets; I don't think adding another derivative makes a difference. (I did think about this when writing the classes and convinced myself it was OK - that the apparent accelerations appear automatically - but I may have fooled myself).